pages/search/index.tsx

```diff
@@ -148,8 +164,20 @@ export default function Search({
               <SkeletonLoader showImage={false} />
             ) : (
               <>
-                {displayAppliedFilters && <AppliedFilters />}
-                <Heading level="h2" mb="xl" size="heading4">
+                {displayAppliedFilters && (
+                  <AppliedFilters aggregations={aggs} />
+                )}
+                <Heading
+                  data-testid="search-results-heading"
+                  level="h2"
+                  // Heading component does not expect tabIndex prop, so we
+                  // are ignoring the typescript error that pops up.
+                  // @ts-expect-error
+                  tabIndex="0"
```

Comment:
You normally would not tab on the heading. For programmatic focus, `-1` is used. Check with Clare on this.

Reply:
updated